src/server/logs/cmds/cmds.go

```diff
@@ -88,6 +88,45 @@ func addDatumRequest(req *logs.GetLogsRequest, datum string) {
 	}
 }
 
+func addPodRequest(req *logs.GetLogsRequest, pod string) {
+	req.Query = &logs.LogQuery{
+		QueryType: &logs.LogQuery_Admin{
+			Admin: &logs.AdminLogQuery{
+				AdminType: &logs.AdminLogQuery_Pod{
+					Pod: pod,
+				},
+			},
+		},
+	}
+}
+
+func addPodContainerRequest(req *logs.GetLogsRequest, pod, container string) {
+	req.Query = &logs.LogQuery{
+		QueryType: &logs.LogQuery_Admin{
+			Admin: &logs.AdminLogQuery{
+				AdminType: &logs.AdminLogQuery_PodContainer{
+					PodContainer: &logs.PodContainer{
+						Container: container,
+						Pod:       pod,
+					},
+				},
+			},
+		},
+	}
+}
+
+func addAppRequest(req *logs.GetLogsRequest, app string) {
+	req.Query = &logs.LogQuery{
+		QueryType: &logs.LogQuery_Admin{
+			Admin: &logs.AdminLogQuery{
+				AdminType: &logs.AdminLogQuery_App{
+					App: app,
```

Comment:
I don’t believe that this is implemented in `logs.go`.

I did implement this as a LogQL query in #9965 (just merged). Do you want to implement the backend, or keep it as client-side LogQL?